sonar/suggestions/rest.py

```
@@ -41,7 +41,7 @@ def completion():
     try:
         service = sonar.service(resource)
         search = service.config.search.search_cls(index=resource)
-    except Exception as err:
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)